lib/agents/engineer.ts

```diff
@@ -14,34 +14,63 @@ import {isDev} from '~/utils/index'
 
 export async function engineer({
 	task,
+	runId,
 	repoFullName,
 	issueNumber,
 	customerId,
 	projectId
 }: {
 	task: string
+	runId: string
 	repoFullName: string
 	issueNumber: number
 	customerId: string
 	projectId: string
 }) {
-	let tokens = {
-		prompt: 0,
-		completion: 0
-	}
-
+	let logId: string
 	const model = new ChatOpenAI({
-		modelName: 'gpt-4-1106-preview',
+		modelName: 'gpt-4-turbo-preview',
```

Comment:
Have you tested the new model thoroughly? I tried it earlier and found weird results from it taking instructions too literally

Reply:
My tests with it for engineering and labeling went pretty well. Was super consistent for me. It should be the same model as gpt-4-1106